src/app/icd/client/DefaultICDClientStorage.h

```diff
@@ -0,0 +1,131 @@
+/*
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "ICDClientStorage.h"
+#include <lib/core/CHIPCore.h>
+
+#include <crypto/CHIPCryptoPAL.h>
+#include <crypto/SessionKeystore.h>
+#include <lib/core/CHIPConfig.h>
+#include <lib/core/CHIPPersistentStorageDelegate.h>
+#include <lib/core/DataModelTypes.h>
+#include <lib/core/ScopedNodeId.h>
+#include <lib/core/TLV.h>
+#include <lib/support/Pool.h>
+#include <vector>
+
+// TODO: SymmetricKeystore is an alias for SessionKeystore, replace the below when sdk supports SymmetricKeystore
+namespace chip {
+namespace Crypto {
+using SymmetricKeystore = SessionKeystore;
+} // namespace Crypto
+} // namespace chip
+
+namespace chip {
+namespace app {
+
+/**
+ * An DefaultICDClientStorage using ICDClientStorage as its backend.
```

Comment:
```suggestion
 * A DefaultICDClientStorage implementation of ICDClientStorage.
```
It's not using ICDClientStorage as a "backend"....